mxcube3/core/components/user/usermanager.py

```diff
@@ -297,10 +312,11 @@ def db_create_user(self, user, password, lims_data):
                 session_id=sid,
                 selected_proposal=user,
                 limsdata=json.dumps(lims_data),
-                roles=["staff"],
+                roles=self._get_configured_roles(user)
```

Comment:
`set` does not support item assignment if I am not wrong. 
Looking for a solution ...

Reply:
It was the idea we don't want to add roles on the fly, is it causing problems for you. We can always change it to a list if needed ?